figures/pipeline/course_daily_metrics.py

```diff
@@ -345,10 +345,13 @@ def load(self, date_for=None, force_update=False, **_kwargs):
         course daily metrics model instance
         """
         if not date_for:
```

Comment:
Yes, because we need to make sure the data collected for a calendar date are complete, so when it is "today" we collect for "yesterday"

I'm strongly thinking of mandating that the course_daily_metrics and site_daily_metrics collectors take in a `date_for` parameter that the caller assigns and doesn't automatically assign to 'yesterday' if the `date_for` is not provided. This puts a bit more work on the caller, but _might_ remove some confusion. But it might not, so I need to think about it